themes/base/src/privacy-manager/components/privacy-manager.tsx

```diff
@@ -12,13 +12,30 @@ import { PRIVACY_MANAGER_DATA_STRUCTURE_VERSION } from '../config'
 
 import PrivacyManagerContext from '../context'
 
+
+interface PrivacyManagerComponentStaticAnnotation {
+  id: string
+  category: string
+  trackingConfig: {}
+}
+
+interface PrivacyManagerConfig {
+  integrations: {},
+  privacyPolicyId: string
+  trackingConfig: {}
+}
```

Comment:
@axe312ger your input is needed here please. Don't know exactly whether both types could be combined / somehow depend on each other and the nested shapes need to be defined.